internal/testutil/gnmi.go

```diff
@@ -29,8 +30,9 @@ import (
 
 // FakeGNMI is a running fake GNMI server.
 type FakeGNMI struct {
-	agent *gnmi.Agent
-	stub  *Stubber
+	agent      *gnmi.Agent
+	stub       *Stubber
+	getWrapper *getWrapper
```

Comment:
I know this was the Ondatra naming, but maybe "clientWrapper" would be a better name, since it wraps the client? Or perhaps the field could be named "clientWrapper" and the struct named "clientWithGetter"